src/lib/config.js

```diff
@@ -63,7 +63,41 @@ function parseAdminEnv () {
   }
 }
 
-function validateLocalEnvConfig () {
+function parseLedgers () {
+  // List of ledgers this connector has accounts on (used to auto-generate pairs)
+  // e.g. ["USD@http://usd-ledger.example","EUR@http://eur-ledger.example/some/path"]
+  return JSON.parse(Config.getEnv(envPrefix, 'LEDGERS') || '[]').map((ledger) => {
+    const sep = ledger.indexOf('@')
+    return { currency: ledger.substr(0, sep), ledger: ledger.substr(sep + 1) }
+  })
+}
+
+function parseNotificationSignEnv () {
+  // On by default in prod
+  const isProduction = process.env.NODE_ENV === 'production'
```

Comment:
Sure. The `loadConfig` exported function could take an object that overrides that. That can be a followup refactor PR